client/components/site-stats-sticky-link/index.jsx

```diff
@@ -12,6 +13,8 @@ var SiteStatsStickyLink = React.createClass( {
 		onClick: React.PropTypes.func
 	},
 
+	_preloaded: false,
```

Comment:
I left that verbatim as found in #1561. For now, I focused on `server/bundler/loader.js` exclusively.